src/transformation/visitors/binary-expression/index.ts

```diff
@@ -45,13 +46,18 @@ export function transformBinaryOperation(
     context: TransformationContext,
     left: lua.Expression,
     right: lua.Expression,
-    operator: BitOperator | SimpleOperator,
+    operator: BitOperator | SimpleOperator | ts.SyntaxKind.QuestionQuestionToken,
     node: ts.Node
 ): lua.Expression {
     if (isBitOperator(operator)) {
         return transformBinaryBitOperation(context, node, left, right, operator);
     }
 
+    if (operator === ts.SyntaxKind.QuestionQuestionToken) {
+        assert(ts.isBinaryExpression(node));
+        return transformNullishCoalescingExpression(context, node);
+    }
```

Comment:
Do we need this branch? If it's required, we can remove `case ts.SyntaxKind.QuestionQuestionToken` from `transformBinaryExpression`

Reply:
Yep you're right, it is needed here, I removed the case from transformBinaryExpression.